ncm-opennebula/src/main/pan/components/opennebula/schema.pan

```diff
@@ -9,6 +9,7 @@ include 'quattor/schema';
 include 'pan/types';
 
 type uuid = string with match(SELF,'[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}');
```

Comment:
probably yes, as far as I know uuid type is used by ceph and opennebula components